docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -92,49 +277,78 @@ See [provider contracts](../contracts/overview.md) for more details.
 
-### Other
+## Suggested changes for providers
+
+- We highly recommend providers to start using to Cluster API v1beta2 types when bumping to CAPI v1.11. 
+  This requires changing following import:
 
-### Suggested changes for providers
+  ```go
+  import (
+      ...
+      clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+  )
+  ```
+  
+  into:
+  
+  ```go
+  import (
+      ...
+      clusterv1 "sigs.k8s.io/cluster-api/api/v1beta2"
+  )
+  ```
+
+  Please refer to [API Changes](#api-changes) for more details about the changes introduced by this release.
+
+  Note: it is technically possible for providers to keep using v1beta1 types from CAPI v1.11, but this is 
+  not recommanded (this approach was not tested during the implementation, we don't know if there are road blockers). 
+
+- If providers are using Conditions from Cluster API resources, we highly recommend providers to check new 
+  conditions instad of old ones. Utils for working with new conditions ara available in the `sigs.k8s.io/cluster-api/util/conditions`.
 
 - We highly recommend providers to start planning the move to the new v1beta2 version of the Cluster API contract;
   the transition MUST be completed before compatibility support for the v1beta1 version of the Cluster API 
   contract will be removed tentatively in August 2026
 
+- Depending on which option you are choosing for the Cluster API contract your are going to implement, please refer to:
+  * [How to bump to CAPI V1.11 but keep implementing the deprecated v1beta1 contract](#how-to-bump-to-capi-v111-but-keep-implementing-the-deprecated-v1beta1-contract)
+  * [How to implement the new v1beta2 contract](#how-to-implement-the-new-v1beta2-contract)
+  
 - We highly recommend providers to define their future strategy for condition management; also in this case the 
   transition to the new condition management strategy MUST be completed before compatibility support for the v1beta1
   version of the Cluster API contract will be removed tentatively in August 2026
   Available options are:
   - Migrate to `metav1.Conditions` like Cluster API (recommended)
   - Replace Cluster API's v1beta1 Conditions with a custom condition implementation that is compliant with
     what is required by the v1beta2 Cluster API contract.  
-    - Starting from the CAPI release when v1beta1 removal will happen (tentative August 2026), the Cluster API project 
-      will remove the Cluster API condition type, the `util/conditions/deprecated/v1beta1` package, the code handling conditions in `util/patch.Helper` 
-      and everything related to the custom Cluster API `v1beta.Condition` type. 
-
-Please refer to following how to guides for additional details.
+    - Starting from the CAPI release when v1beta1 removal will happen (tentative August 2026), the Cluster API project
+      will remove the Cluster API condition type, the `util/conditions/deprecated/v1beta1` package, the `util/deprecated/v1beta1` package,
```

Comment:
Please check for the findings in the similar sentence above

Reply:
Mostly just that there is a similar sentence in a few places and I added 1-2 comments at one of these occurences.

I'll check/compare during the next review